test/pylib/scylla_cluster.py

```diff
@@ -295,19 +296,18 @@ async def cql_is_up(self) -> bool:
             caslog.setLevel(oldlevel)
         # Any other exception may indicate a problem, and is passed to the caller.
 
-    async def rest_api_is_up(self) -> bool:
-        """Test that the Scylla REST API is serving. Can be used as a
-        checker function at start up."""
+    async def get_host_id(self, api: ScyllaRESTAPIClient) -> bool:
+        """Try to get the host id (also tests Scylla REST API is serving)"""
         try:
-            async with aiohttp.ClientSession() as session:
-                url = f"http://{self.hostname}:10000/"
-                async with session.get(url):
-                    return True
-        except aiohttp.ClientConnectionError:
+            self.host_id = await api.get_host_id(self.hostname)
+            return True
+        except (aiohttp.ClientError, HTTPError, RuntimeError) as exc:
```

Comment:
I thought we agreed to **replace** `RuntimeError` with `HTTPError`, not catch them both. `RuntimeError`s should pass through.

I hope it should no longer matter though, because we no longer throw `RuntimeError` from `get_host_id` (IIUC).

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/88d45d24e55c1ac231e596be0fa3284984b9a858